monitoring/mock_uss/start.sh

```diff
@@ -20,6 +20,6 @@ cp health_check.sh /app
 # Start mock_uss server on port 5000
 gunicorn \
     --preload \
-    --workers=4 \
+    --workers=1 \
```

Comment:
More than 1 worker are needed for certain operations; why would we want to reduce to 1 here?